v4/weaviate/graphql/gql.go

```diff
@@ -34,6 +34,11 @@ func (api *API) Aggregate() *AggregateBuilder {
 	return &AggregateBuilder{connection: api.connection}
 }
 
+// RawGraphQLQueryBuilder creates a raw GraphQL query
+func (api *API) Raw(query string) *RawGraphQLQueryBuilder {
+	return &RawGraphQLQueryBuilder{connection: api.connection, query: query}
```

Comment:
could you please rename struct to `Raw` (eventually `RawBuilder`, though there is not much to be build) to match other clients (`GetBuilder`, `Explore`, `AggregateBuilder`)? and filename to match new name.

Reply:
query should be added by `WithQuery` method, not given in the `constructor` to be consistent with other clients

Reply:
Good point to make it concise with other clients, so that the `Raw` query invocation would look similar to other clients, like this:
```
client.GraphQL().Raw().WithQuery(...).Do()
```